package.json

```diff
@@ -16,7 +16,8 @@
   "scripts": {
     "start": "node index.js",
     "dev": "env-cmd -f .env node index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+	"prettysrc" : "npx prettier --write vacabot/**/*.js"
```

Comment:
Wrong indentation here.
And one suggestion, just `prettier` as the command can work.

Reply:
Indentation is fixed.
Please use a verb as a command like `prettify`.

Reply:
Done. Please check now.